gateway/api/proxymanager/proxymanager.go

```diff
@@ -139,6 +140,12 @@ func Post(c *gin.Context) {
 			break
 		}
 
+		hasReview := false
+
+		if pkt.Payload != nil {
+			hasReview = true
+		}
```

Comment:
You don't need to do this, the packet type `pbclient.SessionOpenWaitingApproval` means it has a review.